pkg/app/metadata.json

```diff
@@ -0,0 +1,963 @@
+    },
+    "search": {
+      "apis": ["https://developer.fastly.com/reference/api/services/service/#search-service"]
+    },
+    "update": {
+      "apis": ["https://developer.fastly.com/reference/api/services/service/#update-service"]
+    }
+  },
+  "service-version": {
+    "activate": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#activate-service-version"]
+    },
+    "clone": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#clone-service-version"]
+    },
+    "deactivate": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#deactivate-service-version"]
+    },
+    "list": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#list-service-versions"]
+    },
+    "lock": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#lock-service-version"]
+    },
+    "update": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#update-service-version"]
+    }
+  },
+  "stats": {
+    "historical": {
+      "apis": ["https://developer.fastly.com/reference/api/metrics-stats/historical-stats/#get-hist-stats-service"]
+    },
+    "realtime": {
+      "apis": ["https://developer.fastly.com/reference/api/metrics-stats/realtime/#get-stats-last-second"]
+    },
+    "regional": {
+      "apis": ["https://developer.fastly.com/reference/api/metrics-stats/historical-stats/#get-regions"]
+    }
+  },
+  "update": {
+  },
+  "user": {
+    "create": {
+      "apis": ["https://developer.fastly.com/reference/api/account/user/#create-user"]
+    },
+    "delete": {
+      "apis": ["https://developer.fastly.com/reference/api/account/user/#delete-user"]
+    },
+    "describe": {
+      "apis": [
+        "https://developer.fastly.com/reference/api/account/user/#get-current-user",
+        "https://developer.fastly.com/reference/api/account/user/#get-user"
+      ]
+    },
+    "list": {
+      "apis": ["https://developer.fastly.com/reference/api/account/customer/#list-users"]
+    },
+    "update": {
+      "apis": [
+        "https://developer.fastly.com/reference/api/account/user/#request-password-reset",
+        "https://developer.fastly.com/reference/api/account/user/#update-user"
+      ]
+    }
+  },
+  "vcl": {
+    "custom": {
+      "create": {
+        "apis": ["https://developer.fastly.com/reference/api/vcl-services/vcl/#create-custom-vcl"]
+      },
+      "delete": {
+        "apis": ["https://developer.fastly.com/reference/api/vcl-services/vcl/#delete-custom-vcl"]
+      },
+      "describe": {
+        "apis": ["https://developer.fastly.com/reference/api/vcl-services/vcl/#get-custom-vcl"]
+      },
+      "list": {
+        "apis": ["https://developer.fastly.com/reference/api/vcl-services/vcl/#list-custom-vcl"]
+      },
+      "update": {
+        "apis": ["https://developer.fastly.com/reference/api/vcl-services/vcl/#update-custom-vcl"]
+      }
+    },
+    "snippet": {
```

Comment:
This demonstrates some useful flags like `--content`, and in some cases the descriptions reflect on the changes necessary to move between a 'dynamic' snippet and a 'versioned' snippet.